testing/vcs/test_lon_axes_freak_out.py

```diff
@@ -0,0 +1,29 @@
+import os,sys,vcs,cdms2
+src=sys.argv[1]
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+
+f=cdms2.open(os.path.join(sys.prefix,"sample_data","clt.nc"))
```

Comment:
@dlonie good idea, we should create an issue to update all tests I'm sure it's everywhere.